apple-pie/styles/style.css

```diff
@@ -0,0 +1,69 @@
+html { 
+ margin: 0;
+ padding: 0;
+ font-family: 'Open Sans', sans-serif;
+ font-size: 18px
+}
+body { 
+    margin: 0;
+    padding: 0;
+   }
+   p {
+    margin:0;
+    padding:0;
+}
+.theApple {
+    display: flex;
+    background-image: url("../images/apple-pie.jpg");
+    background-position: center;
+    background-size: cover;
+    height: 460px;
+    width: 100vw;
+	justify-content: center;
+	align-items: center;
+}
+.theApple p {
+    font-size: 80px;
+    color :white;
+}
+.container {
+    display: flex;
+	flex-direction: column;
+    margin-left: 10%;
+    margin-right: 10%;
+    margin-top: 3%;
+    margin-bottom: 5%;
+}
+.container img {
+    height: 100px;
```

Comment:
you can't hardcoded width and height, you can do one only, otherwise, the image is squeezed because you didn't fallow the image proportions, images should be centred as well, I recommend:
```
.container img {
    width: 450px;   
    margin-top: 5%;
    margin-left: 25%;
}
```